### PR TITLE
Add modifier support to import

### PR DESCRIPTION
**Pull request by robertfoss**
Support using modifiers with the new GBM call GBM_BO_IMPORT_FD_MODIFIER.

### CODE REVIEW

File: gralloc_gbm.cpp [hunk 1]
```diff
@@ -133,14 +137,18 @@ static struct gralloc_gbm_bo_t *gbm_import(struct gbm_device *gbm,
 	data.height = handle->height;
 	data.stride = handle->stride;
 	data.format = format;
-
```
Comment (robherring) [inline gralloc_gbm.cpp:136]: Stray WS change

**Review by robherring — COMMENTED** [review]
Looks good to me other than one nit.

File: gralloc_gbm.cpp (hunk 1 above)
Comment (robertfoss) [reply gralloc_gbm.cpp:136]: Fix pushed.

**Comment by robherring** [comment]
Merged.

**Pull request merged**